pkg/process/model/message_test.go

```diff
@@ -0,0 +1,31 @@
+package model
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestDecodeZstd05Payload ensures backward compatibility with our intake
+func TestDecodeZstd05Payload(t *testing.T) {
+	file := "./test_zstd.0.5.dump"
```

Comment:
There is a convention for this, [`testdata`](https://dave.cheney.net/2016/05/10/test-fixtures-in-go)

Reply:
👍